cla_backend/libs/eligibility_calculator/cfe_civil/dependants.py

```diff
@@ -0,0 +1,22 @@
+import datetime
+
+
+def translate_dependants(todays_date, facts):
+    dependants = []
+    for child in range(facts.dependants_young):
+        dependants.append(
+            dict(date_of_birth=str(datetime.date(todays_date.year - 15, todays_date.month, todays_date.day)),
+                 in_full_time_education=False,
+                 relationship="child_relative",
+                 income=dict(frequency="weekly", amount=0),
+                 assets_value=0),
+        )
+    for adult in range(facts.dependants_old):
+        dependants.append(
+            dict(date_of_birth=str(datetime.date(todays_date.year - 17, todays_date.month, todays_date.day)),
+                 in_full_time_education=False,
+                 relationship="adult_relative",
```

Comment:
We are assuming 17&18yo are "adult_relative" - can we check the implications of this in the calcs?